.github/workflows/cicd.yml

```diff
@@ -0,0 +1,78 @@
+name: CI/CD
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
```

Comment:
We have to align the tags with other repositories. Double check this with Jimmy.

Reply:
@lluisFtrack do you have any other repo to point me to so I can compare ? 
Thanks.